meshtastic/node.py

```
@@ -998,25 +1033,30 @@ def setURL(self, url: str, addOnly: bool = False) -> None:
+                for disabled_channel, new_settings in zip(
+                    disabled_channels, pending_new_settings
+                ):
+                    disabled_channel.settings.CopyFrom(new_settings)
+                    disabled_channel.role = channel_pb2.Channel.Role.SECONDARY
+                    channels_to_write.append(
+                        (disabled_channel.index, new_settings.name, new_settings)
+                    )
 
             for ignored_name in ignored_channel_names:
                 logger.info(
                     f'Ignoring existing or empty channel "{ignored_name}" from add URL'
```

Ruff (0.15.5)

[warning] Logging statement uses f-string

(G004)